### ISSUE TITLE
"JavaScript > Suggest : Auto Imports" setting description looks incorrect

### ISSUE THREAD

**Issue by robole** [microsoft/vscode#104178]
- VSCode Version:1.48.0-insiders
- OS Version: Windows_NT x64 10.0.18362

Steps to Reproduce:

1. Open Settings (File > Preferences > Settings)
2. Find the setting  "JavaScript > Suggest : Auto Imports"
3. Read description! 

The description is "Enable/disable auto import suggestions. Requires using Typescript 2.6.1 or newer in workspace." Surely, there is no need for Typescript in the workspace if it is a JavaScript-related setting?

The same applies to "JavaScript > Update Imports on File Move: Enabled".

**Comment by IllusionMH** [comment]
Language features for both TS and JS are powered by TS. This info is relevant if you change what TS version VS Code should use for current workspace (may not be located directly in workspace).
And especially relevant to new feature where you most likely need to use `typescript@next` to get those features in JS files.

---

On the other hand built-in version in insiders in 3.9.7 and anything below 3.0 is not supported on DefinitelyTyped due to low usage
> Telemetry indicates 99.9% of VS Code users are 3.0 or newer
Telemetry indicates 97% of VS users are on 3.0 or newer

[Source](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/40050#issue-515633467)

So it make sense to remove mentions of minimum TS versions where they are older than 3.0

**Comment by mjbvz** [comment]
Thanks @IllusionMH!

Removing the mention of old TS version makes sense, especially for JS specific settings. Marking as a good first issue

**Comment by epiacentini** [comment]
Hi I am new to open-source contributing and the codebase is very large where would I look to try and solve this problem?

**Comment by IllusionMH** [comment]
@epiacentini you should check https://github.com/Microsoft/vscode/wiki/How-to-Contribute for Prerequisites and instruction how to run in from source so you'll be able to verify your changes.

Most likely all strings that need to be changes are stored in [typescript-language-features/package.nls.json](https://github.com/microsoft/vscode/blob/master/extensions/typescript-language-features/package.nls.json) but make sure to run search yourself if I messed something.

**Comment by epiacentini** [comment]
Ok great thank you. Also this may be a separate issue but when trying to follow the recreation steps i dont have preferences under File, its under the code section. Might be because i am using an earlier version 1.41.1, however i cant update my vscode because im on a "read-only-volume"

**Comment by epiacentini** [comment]
Nevermind I figured it out

**Comment by epiacentini** [comment]
I submitted my PR but it failed one of the checks and i'm not sure why. This is the link to the PR: https://github.com/microsoft/vscode/pull/104207

**Comment by robole** [comment]
Thanks. You can add these to the list then!

`TypeScript > Suggest: Auto Imports: Enable/disable auto import suggestions. Requires using Typescript 2.6.1 or newer in workspace.`
`TypeScript > Update Imports on File Move: Enabled: Enable/disable automatic updating of import paths when you rename or move a file in VS Code. Require using TypeScript 2.9 or newer in the workspace.`

**Comment by KDharmindar** [comment]
Hi @robole @mjbvz and @epiacentini 
Is this issue still open? I would like to fix the issue. I am a new comer and want to become a full time contributor.

**Comment by epiacentini** [comment]
I submitted my PR but I am just waiting on it to get approved. @KDharmindar

**Comment by KDharmindar** [comment]
ok @epiacentini . I will pick something else. I am trying to build it through source. Have you been able to do that?

**Comment by mjbvz** [comment]
Fixed by #104211
